site/contents/write_ups/2021/03-11-21_writeup.md

```diff
@@ -0,0 +1,109 @@
+
+Discussing what a suitable regulatory body would look like led to us considering the dichotomy between accountability and efficiency. If the ethics board was managed by a single person, that person is extremely susceptibile to letting their bias affects outcomes. The way to get around this is to have ethics committees with a variety of viewpoints. Burearcracy inevitably slows down the government massively and this can get in the way of innovation. 
+
+Perhaps a workaround is open source committees managed by trusted communities who can respond faster, but the UK government (and indeed, most governments) would likely be reluctant to give up their governing responsibilities to such an extent. 
+
+Another solution we discussed was the ombudsman approach to regulation - a figure is given wide authority in regulatory decision-making, essentially asked to exercise judgement within a broad set of parameters. Typically they are drawn from industry, or at least have some deep understanding of the industry, so that their judgements can be well-founded. The concept originated in Sweden in 1809, with ombudsman coming from the Swedish for "legal representative". 
+
+It has been adopted it in some UK markets ("ombudsman" often used synonymously with "Parliamentary Commissioner for Administration") but not always successfully. When you have an industry that is moving so quickly it is difficult (if not impossible) to have a regulatory system that can cope if you try to write down lots of rules. An ombudsman approach allows for greater speed of reaction and flexibility.
+
+(As a quick note, ombudsman is used a gender neutral term, as the suffix "-man" is a gender neutral suffix in the Swedish language. Regardless, the term has fallen out of usage in recent years.)
+
+How will the UK approach this problem across different sectors? Would there be a national/ethical committee for every industry? It sounds to us like a lot of hoops to jump through before we reach a solution.
+
+We saw issue with a seeming dispraity between public and private sector regulation. Whilst the report outlined the public sector restrictions, it didn't seem like there would be any regulations in the private sector which is troubling but frankly not surprising.
+
+What do the authors mean by innovation? Some of us feared a barrage of "new technologies" which will simply be a collection of linear regression soups garnished with a few "for" and "if" loops?
+
+## How do you see regulation as potentially impacting your work? If you are not UK based, are there similarities/differences with any of your national regulations?
+
+Is the European approach better? Does it give us more confidence? They have guidelines from the EU.
```

Comment:
```suggestion
```

Suggest removing this sentence - I found it confusing as an intro to this section and think you cover the EU side later on :)